config/notifiers.go

```diff
@@ -163,6 +163,9 @@ type EmailConfig struct {
 	HTML         string            `yaml:"html,omitempty" json:"html,omitempty"`
 	Text         string            `yaml:"text,omitempty" json:"text,omitempty"`
 	RequireTLS   *bool             `yaml:"require_tls,omitempty" json:"require_tls,omitempty"`
+	TLSCA        string            `yaml:"tls_ca,omitempty" json:"tls_ca,omitempty"`
```

Comment:
Instead of injecting these into the `EmailConfig` struct, I think @brian-brazil was referring to use [TLSConfig](https://github.com/prometheus/alertmanager/blob/master/vendor/github.com/prometheus/common/config/http_config.go#L291) from the common package.

Reply:
Just understand, you want I use the TLSConfig struct instead of 3 strings in EmailConfig

but server name will be redundant

https://github.com/prometheus/alertmanager/blob/master/vendor/github.com/prometheus/common/config/http_config.go#L300

Reply:
I don't see why server_name would be redundant, the DNS name you're using may not match the tls cert the server itself knows about.